Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using hardcoded passwords like '1234' is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.